Mathlib/Data/NNRat/Defs.lean

```diff
@@ -147,11 +147,11 @@ theorem coe_eq_zero : (q : ℚ) = 0 ↔ q = 0 := by norm_cast
 theorem coe_ne_zero : (q : ℚ) ≠ 0 ↔ q ≠ 0 :=
   coe_eq_zero.not
 
-@[norm_cast] -- Porting note (#10618): simp can prove this
+@[norm_cast]
```

Comment:
It doesn't look like you added `@[simp]` here

Reply:
True. While making #18075 I thought that these could be `@[simp]` but in the end they were provable after some more imports. See https://github.com/leanprover-community/mathlib4/pull/18076/commits/8fad80c0060d6f5d4ff4f431e26c9e563b35b549. But by then #18075 had already been merged.

Would you like me to move these changes to a separate cleanup PR?